School.js

```diff
@@ -43,7 +65,15 @@ class School {
    */
   getStudentsByClass(className) {
     // Your code here
+    if(this.classes(className)){
```

Comment:
It should be `this.classes[className]` the way you have it seem like classes is a function and className is the argument.